.github/workflows/build-test.yml

```diff
@@ -103,3 +103,12 @@ jobs:
       - name: Build sources
         run: |
           cmake -Bbuild -Hiceoryx_meta -DBUILD_TEST=ON -DINTROSPECTION=OFF -DBINDING_C=ON -DEXAMPLES=ON && cmake --build build
```

Comment:
You can add here `-DCMAKE_CXX_FLAGS="/MP"` to build windows with multiple CPU cores

Reply:
I already added this cxx flag in iceoryx_hoofs/cmake/IceoryxPlatform.cmake, line 37